doc/source/migrations/migrate0_1.rst

```diff
@@ -47,17 +47,21 @@ and will be removed in MontePy 1.0.0.
 New Interface & Migration
 -------------------------
 
+For more details see the new :ref:`mat_tutorial` tutorial in the getting started guide,
+as well as the example in the :class:`~montepy.data_inputs.material.Material` documentation.
+
 .. note::
 
         This design is not finalized and is subject to change.
         This is the currently planned design for ``1.0.0a1``.
         If you have input you can `join the discussion <https://github.com/idaholab/MontePy/discussions/475>`_.
-        This is also where alpha-testing will be announced.
+        For feedback on the alpha test please `join this discussion <https://github.com/idaholab/MontePy/discussions/604>`_.
```

Comment:
comma after "alpha test"

I'll push a commit with the comma usage

Reply:
You would get along with my 10th grade english teacher.